docs/blog/newsletter-november-2020.rst

```diff
@@ -0,0 +1,141 @@
+
+To learn more about creating accessible websites:
+
+- A website that lets you test for accessibility components on web pages, `Accessibility Support <https://a11ysupport.io/>`_
+- The `Web Accessibility Initiative <https://www.w3.org/WAI/>`_
+- The `POUR principles <https://webaim.org/articles/pour/>`_ for creating a truly accessible website (Perceivable, Operable, Understandable, Robust)
+
+For those who want a deeper dive:
+
+- `Certification requirements <https://www.accessibilityassociation.org/certification>`_ from the International Association of Accessibility Professionals
+- `Introduction to Web Accessibility <https://www.edx.org/course/web-accessibility-introduction>`_ from edX
+- `The Paciello Group <https://www.paciellogroup.com>`_, an accessibility consulting firm, provide services and resources 
+- `Classes from Deque <https://dequeuniversity.com/curriculum/online-classes/>`_, another accessibility consulting firm
+
+For a real-world tale of why accessibility matters, see `5 most annoying website features I face as a blind person every single day <https://bighack.org/5-most-annoying-website-features-i-face-as-a-blind-screen-reader-user-accessibility/>`_.
+
+Those of us who document APIs should consider that `API Document Generators Can Fall Woefully Short on Meeting Accessibility Guidelines <https://thenewstack.io/api-document-generators-can-fall-woefully-short-meeting-accessibility-guidelines/>`_. The article is three years old, but the problem remains, although one WTDer called out `ReDoc <https://github.com/Redocly/redoc>`_ as a good option for its attention to accessibility issues.
+
+Members of the community also gave shoutouts to resources for bias-free language:
+
+- `Bias-Free Language <https://apastyle.apa.org/style-grammar-guidelines/bias-free-language/>`_ in the APA Style Guide
+- `Terminology, Power and Oppressive Language <https://tools.ietf.org/id/draft-knodel-terminology-00.html>`_, a draft standard from the IETF
+
+Finally, a few handy WTD resources: `lists of resources for inclusive writing and accessibility tooling <https://www.writethedocs.org/guide/writing/style-guides/#reducing-bias-in-your-writing>`_, plus `some relevant conference talks and writeups in newsletters </topics/#accessibility-and-inclusivity>`__. 
+
+------------------------------------------
+Dealing with stress & struggling to write
+------------------------------------------
+
+Is it any surprise that we've been talking a lot about stress recently? Truly working out how to deal with stress is so individual, and means looking at the big picture - it's not something that you can sort out with one or two quick tips. But our community members recently shared some thoughts on how they manage stress, especially when that stress causes you to struggle to focus and complete work. Maybe some of them you'll find helpful.
```

Comment:
I wonder if "Maybe some of them you'll find helpful" might be a difficult construction. I like it but it seems almost like a regional thing that someone unfamiliar could stumble over -- "Maybe you'll find some of them helpful" seems more straightforward.

Overthinking as usual here... :D

Reply:
Good thinking! :+1: 

> Maybe some of them will work for you

could be another option.

Reply:
Not overthinking, it's awkwardly phrased. Fixed as Hillary suggested.